linode/data_source_linode_stackscript_test.go

```diff
@@ -0,0 +1,45 @@
+package linode
+
+import (
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/helper/resource"
+)
+
+func TestAccDataSourceLinodeStackscript_basic(t *testing.T) {
+	t.Parallel()
+
+	resourceName := "data.linode_stackscript.stackscript"
+
+	resource.Test(t, resource.TestCase{
+		PreCheck:  func() { testAccPreCheck(t) },
+		Providers: testAccProviders,
+		Steps: []resource.TestStep{
+			{
+				Config: testDataSourceLinodeStackScriptBasic(),
+				Check: resource.ComposeTestCheckFunc(
+					resource.TestCheckResourceAttr(resourceName, "label", "my_stackscript"),
+					resource.TestCheckResourceAttr(resourceName, "description", "test"),
+					resource.TestCheckResourceAttr(resourceName, "script", "#!/bin/bash\necho hello world\n"),
+				),
```

Comment:
Images is also set so you can check for that too. There are other fields whose setters would be cheap to test as well but it will require more stackscript parameters in the fixture.